radis/lbl/broadening.py

```diff
@@ -800,6 +802,21 @@ def _calc_broadening_HWHM(self):
         # Init variables
         df = self.df1
 
+        try:
```

Comment:
This try /except is hard to understand. 

What's the error you're expecting? You should write it in except, like except AttributeError or KeyError. 
Add some comments too?

Reply:
SpectrumFactory does not have a df0. So I have put a try/except for that case.

Reply:
It always has df0, unless database wasn't loaded (but then it should fail elsewhere). Can you provide an example?

Reply:
Examples - [1](https://github.com/radis/radis/blob/7c1be93c94ee3f50f03ef362e36a46cb6068f57d/radis/test/io/test_hitemp.py#L80) [2](https://github.com/radis/radis/blob/7c1be93c94ee3f50f03ef362e36a46cb6068f57d/radis/test/io/test_hitemp.py#L97)

Reply:
Back on this one : we shouldn't have a try/except here. Maybe the attributes are not transfered when df1 is created? Can you look it up?

Reply:
I believe the problem is arising because even we are no longer transferring the attributes that are stored as `attrs` from `df0` to `df1` anymore. I can alternatively, remove the try/except and instead retain the code present in the except block alone.